### ISSUE TITLE
Feature Request: Convert steamdeps.txt to a distro-agnostic format

### ISSUE THREAD

**Issue by Tele42** [ValveSoftware/steam-for-linux#1200]
If Steam is to use steamdeps.txt and /usr/bin/steamdeps as the future solution to package manager integration, then the ubuntu-specific package list needs to be replaced by a distro-agnostic format. The adaptor script should be able to do a lookup and translate the list into individual packages, instead of being spoon fed the package names.
The simplest naming scheme would probably be libprimarylibraryname.so.1:32.
If an automated lookup method is not available, it will take the same amount of time to make a translation table from the distro-agnostic set to the distro-specific package list as it would from the ubuntu-specific set, and it is unreasonable to expect game makers to repeatedly revise their dependency list as distributions are officially supported.

To put it another way, a single unified set of dependencies for a package takes less man hours to maintain compared to a dozen distro-specific sets in the same file.

**Comment by cognitus** [comment]
what's state? I had update the client with the last beta and I had edit the script for open 
steam...

**Comment by Tele42** [comment]
Whatever value this request had back in 2013 has effectively dried up and all major distros have manually handled dependencies their own way.
